src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/enable-lambda-monitoring/account-linking.mdx

```diff
@@ -170,10 +171,140 @@ com/secrets-manager/) for greater security.
     ```
+  
+    See also our [legacy docs on how to manually instrument your account](/docs/serverless-function-monitoring/aws-lambda-monitoring/enable-lambda-monitoring/enable-serverless-monitoring-aws-lambda-legacy/). 
 
   </Collapser>
-</CollapserGroup>
 
+  <Collapser
+    id="env-var"
+    title="Lambda environment variables"
+  >
+    
+    For reference, these are the available environment variables for configuring and controlling the module behavior. 
+
+    **Required**:
+
+    <table>
+      <thead>
+        <tr>
+          <th style={{ width: "350px" }}>
+            Environment variable
+          </th>
+
+          <th>
+            Description
+          </th>
+        </tr>
+      </thead>
+
+      <tbody>
+
+        <tr>
+          <td>
+            `NEW_RELIC_ACCOUNT_ID`
+          </td>
+
+          <td>
+            Required for distributed tracing.
+          </td>
+        </tr>
+
+        <tr>
+          <td>
+            `NEW_RELIC_LAMBDA_HANDLER`
+          </td>
+
+          <td>
+            Required for Node, Python, and Java. 
+          </td>
+        </tr>
+        
+        <tr>
+          <td>
+            `NEW_RELIC_TRUSTED_ACCOUNT_KEY`
+          </td>
+
+          <td>
+            Required for distributed tracing. This is your account's ID. If your account is a child account, this is the account ID for the root/parent account.
+          </td>
+        </tr>
+
+      </tbody>
+    </table>
+
+    **Optional**:
+
+    <table>
+      <thead>
+        <tr>
+          <th style={{ width: "350px" }}>
+            Environment variable
+          </th>
+
+          <th>
+            Description
+          </th>
+        </tr>
+      </thead>
+
+      <tbody>
+
+        <tr>
+          <td>
+            `NEW_RELIC_DISTRIBUTED_TRACING_ENABLED`
+          </td>
+
+          <td>
+            Used in [legacy set-ups](/docs/serverless-function-monitoring/aws-lambda-monitoring/enable-lambda-monitoring/enable-serverless-monitoring-aws-lambda-legacy/). `TRUE` by default for most agents.
+          </td>
+        </tr>
+
+        <tr>
+          <td>
+            `NEW_RELIC_EXTENSION_LOG_LEVEL`
+          </td>
+
+          <td>
+            Used for debugging.
+          </td>
+        </tr>
+
+        <tr>
+          <td>
+            `NEW_RELIC_EXTENSION_SEND_FUNCTION_LOGS`
+          </td>
+
+          <td>
+            `FALSE` by default. Set to `TRUE`to send function logs to New Relic directly, bypassing CloudWatch and the `newrelic-log-ingestion` Lambda.
+          </td>
+        </tr>
+
+        <tr>
+          <td>
+            `NEW_RELIC_LAMBDA_EXTENSION_ENABLED`
+          </td>
+
+          <td>
+            TBC
+          </td>
+        </tr>
+
+        <tr>
+          <td>
+            `NEW_RELIC_LICENSE_KEY` / `NEW_RELIC_LICENSE_KEY_SECRET`
+          </td>
+
+          <td>
+            Unless stated otherwise, we use a reasonable default value for the license key secret. 
```

Comment:
If you're going to document these, you might as well actually document them. the LK is the classic NR license key. It's the highest-priority source for the LK. It's not set by default.

The secret is the name of the AWS Managed Secret containing the license key. Assuming the LK var itself is not set, we look the key up there. We have a default for this, which integrates with the CLI tools. 

NB: it's really important that the contents of the LK secret be formatted correctly. We suggest using the CLI to set the secret up, as it does the right thing.

Reply:
Thanks @MattWhelan - I was using the comments in the spreadsheet, which were very confusing, that's why I wanted to check with you. I'll rephrase this, could you have another look?